sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -152,6 +152,37 @@ def _set_rewrite_response_callback(self, callback):
     """
     self._rewrite_cb = callback
 
+  def get_bucket(self, bucket_name):
```

Comment:
Could you update this method to look more like `def exists(...)` below? (add retries, return None on 404 but otherwise raise)